kytos/core/napps/manager.py

```diff
@@ -95,6 +95,12 @@ def install(self, napp_uri, enable=True):
 
         log.info("New NApp installed: %s", napp)
 
+        napp = NApp.create_from_json(dst/'kytos.json')
+        log.info('Installing Napp Dependencies:')
+        if napp.napp_dependencies:
```

Comment:
I think we do not need this "if".
The `napp_dependencies` attribute must always exist. It should always be a list. If there are no dependencies, it should be an empty list.
Therefore, the `for napp in napp.napp_dependencies` will not fail it `napp_dependencies` is an empty list.